frontend/app/src/components/history/events/HistoryEventsView.vue

```diff
@@ -171,6 +178,10 @@ const {
 
       const accounts = get(usedAccounts);
 
+      if (isDefined(location)) {
```

Comment:
I already mentioned in the previous comment that for this to work we have to make the initial value for the prop undefined (or null) for this to work.

if the default value for location is `''` then this would always be true because `''` is a defined value.

Reply:
sorry my bad, should be updated now